mimic-iv/buildmimic/bigquery/schemas/hosp/emar.json

```diff
@@ -1 +1,62 @@
-[{"name": "subject_id", "type": "INT64", "mode": "REQUIRED"}, {"name": "hadm_id", "type": "INT64", "mode": "NULLABLE"}, {"name": "emar_id", "type": "STRING", "mode": "REQUIRED"}, {"name": "emar_seq", "type": "INT64", "mode": "REQUIRED"}, {"name": "poe_id", "type": "STRING", "mode": "REQUIRED"}, {"name": "pharmacy_id", "type": "INT64", "mode": "NULLABLE"}, {"name": "charttime", "type": "DATETIME", "mode": "REQUIRED"}, {"name": "medication", "type": "STRING", "mode": "NULLABLE"}, {"name": "event_txt", "type": "STRING", "mode": "NULLABLE"}, {"name": "scheduletime", "type": "DATETIME", "mode": "NULLABLE"}, {"name": "storetime", "type": "DATETIME", "mode": "REQUIRED"}]
\ No newline at end of file
+[
+    {
+        "name": "subject_id",
+        "type": "INT64",
+        "mode": "REQUIRED"
+    },
+    {
+        "name": "hadm_id",
+        "type": "INT64",
+        "mode": "NULLABLE"
+    },
+    {
+        "name": "emar_id",
+        "type": "STRING",
+        "mode": "REQUIRED"
+    },
+    {
+        "name": "emar_seq",
+        "type": "INT64",
+        "mode": "REQUIRED"
+    },
+    {
+        "name": "poe_id",
+        "type": "STRING",
+        "mode": "REQUIRED"
+    },
+    {
+        "name": "pharmacy_id",
+        "type": "INT64",
+        "mode": "NULLABLE"
+    },
+    {
+        "name": "enter_provider_id",
+        "type": "STRING",
+        "mode": "NULLABLE"
+    },
```

Comment:
@alistairewj , this block didn't appear in the previous version and the variable name seemed a bit odd. Just wanted to point that out in case it was a mistake.

Reply:
yeah this column is new to v2.2